carbon-design-system/carbon
# refactor(react): update ProgressIndicator to FC

Closes #10281

This PR updates `ProgressIndicator` to a functional component

#### Changelog

**New**

- Add `next/ProgressIndicator.js` and corresponding tests

**Changed**

- Toggle on classic vs next ProgressIndicator behind feature flag

**Removed**

#### Testing / Reviewing

- Verify Progress Indicator continues to work in classic storybook
- Verify Progress Indicator works in next storybook
